packages/mephisto-worker-addons/webpack.config.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+var path = require("path");
+var webpack = require("webpack");
+
+module.exports = {
+  entry: "./src/index.jsx",
+  output: {
+    path: __dirname,
+    filename: "lib/index.js",
+    library: "mephisto-worker-addons",
+    libraryTarget: "umd",
+  },
+  target: "web",
+  externals: {
+    react: "react",
+  },
```

Comment:
For some reason adding `react-dom` to this externals object will cause the task that is linking to this package to receive a cannot find `react-dom` error. 

May be worthwhile to investigate this at a different time